src/tutorial/output.md

```diff
@@ -72,11 +72,31 @@ in the [documentation for the `std::fmt` module][std::fmt].
 
 ## Printing errors
 
-<aside class="todo">
+Printing errors should be done via `stderr`
+to make it easier for users
+and other tools
+to pipe their outputs to files
+or more tools.
 
-**TODO:**
-`eprintln!` and stdout vs stderr
-[Issue #66](https://github.com/rust-lang-nursery/cli-wg/issues/66)
+In Rust this is achieved
+with `println!` and `eprintln!`,
+the former printing to `stdout`
+and the latter to `stderr`.
+
+```rust
+println!("This is information");
+eprintln!("This is an error! :(");
+```
+
+<aside>
+
+**Beware**: Printing escape codes can be dangerous,
+putting the user's terinal into a weird state.
+Always be careful when manually printing them!
+
+Ideally you should be using the `ansi_term` crate
```

Comment:
Maybe

> you should be using {--the `ansi_term` crate--}{++a crate like `ansi_term`++}

because I'm not sure it works on Windows, e.g. (I've used termcolor before because of that but with it I actually forgot to call reset inc cargo-fix which was… sad)